polyfill/spatial-navigation-polyfill.js

```diff
@@ -1151,7 +1188,7 @@
     const P1 = Math.abs(points.entryPoint[0] - points.exitPoint[0]);
     const P2 = Math.abs(points.entryPoint[1] - points.exitPoint[1]);
 
-    // A: The euclidian distance between P1 and P2.
+    // A: The euclidean distance between P1 and P2.
     const A = Math.sqrt(Math.pow(P1, 2) + Math.pow(P2, 2));
```

Comment:
How about using `getEuclideanDistance` function here?

```
const A = getEuclideanDistance(rect1, rect2, dir);
```

Reply:
Hmm, this opinion make sense.
But I wanted to separate 3 function independently according to spatial-navigation-function: normal | euclidean | grid.

If you prefer to use getEuclideanDistance() in here, I'll change.

Reply:
I got your point. To calculate the euclidean distance is a well-known feature, so I thought this function could be reusable. 
I don't insist strongly, so please do as you prefer.